src/Tools/ReleaseNotesUtil/FixerExtensions.cs

```diff
@@ -80,7 +80,7 @@ private static bool HasImplementation(CodeFixProvider fixer)
             // {
             //     return Task.CompletedTask;
             // }
-            byte[] methodBodyIL = method?.GetMethodBody()?.GetILAsByteArray();
+            byte[]? methodBodyIL = method?.GetMethodBody()?.GetILAsByteArray();
             if (methodBodyIL != null
```

Comment:
💡 I believe the `method!` suppression below can be removed if you rewrite the condition here.

```suggestion
            if (method?.GetMethodBody()?.GetILAsByteArray() is { } methodBodyIL
```